frontend/src/hooks/useFields.ts

```diff
@@ -96,6 +96,7 @@ export function useFields<T extends Zaak = Zaak>(
     {
       name: "startdatum",
       type: "daterange",
+      filterLookup: "startdatum__gte,startdatum__lte",
```

Comment:
Why is this? Without the filter is: `?startdatum__gte=0001-01-01&startdatum__lte=0002-02-02`, but with it becomes: `?startdatum__gte%2Cstartdatum__lte=Invalid+Date%2CInvalid+Date`, I don't think thats correct?